lib/pbench/server/cache_manager.py

```diff
@@ -691,11 +562,11 @@ def delete_if_empty(directory: Path) -> None:
             directory.rmdir()
 
     def __init__(self, options: PbenchServerConfig, logger: Logger):
-        """
-        Construct a CacheManager object. We don't do any discovery here, because
-        the mutation operations allow dynamic minimal discovery to save on
-        some time. The `full_discovery` method allows full discovery when
-        desired.
+        """Construct a CacheManager object.
+
+        We don't do any discovery here, because the mutation operations allow
+        dynamic minimal discovery to save on some time. The `full_discovery`
```

Comment:
I realize that you didn't actually change this, but I would drop either "on" or "some" -- having them both doesn't improve over using just one of them.

Reply:
Well of course technically I *did* change this, even if I didn't bother to read it carefully when I broke off a summary and re-wrapped it, so fair game. 😦